### ISSUE TITLE
Lengths of inputs and outputs

### ISSUE THREAD

**Issue by yulonglin** [tau-nlp/scrolls#2]
1. From the paper, you seem to truncate inputs to 16,384 tokens for your leaderboard, is that right?
2. As n-gram metrics are affected by the length of outputs, how do you determine the target length of outputs? I notice that the default max_target_length in baselines/src/run.py is 128 tokens. Do you train your models with an EOS token such that the generated output may terminate much earlier?

**Comment by UriSha** [comment]
Hi,

Thanks for your questions.

1. We do not truncate input for the leaderboard. Since truncating is the most common method for cases when there are more input tokens than what models can process, we ask people who make submissions to mention what is their models max input tokens capacity.
2. We evaluate submissions against the full reference strings, without any truncation. We did train with EOS, but not up to 128 tokens. We used the max output tokens that the models were pretrained to. See https://github.com/tau-nlp/scrolls/blob/main/baselines/scripts/commands/finetune.py#L46

I hope that helps.
